opt_base_2_bytes.go

```diff
@@ -46,6 +46,14 @@ func (o OptBase2Bytes) IsDefined() bool {
 	return o.size != nil
 }
 
+func (o OptBase2Bytes) GetOrElse(orElseValue units.Base2Bytes) units.Base2Bytes {
```

Comment:
Side note: would it mess things up to have the constructor and all the methods operate on a pointer receiver? Since the struct itself contains a pointer, it might be confusing that if you copy it, the copy isn't actually independent.

Reply:
I have updated it where I could.

Most methods have to operate as they are defined so they conform to the SingleValue interface, which precludes the use of pointer receivers.